addons/core/addon/services/scope.js

```diff
@@ -13,8 +13,18 @@ export default class ScopeService extends Service {
    */
   @tracked org;
 
+  /**
+   * @type {ScopeModel}
+   */
+  @tracked orgsList;
```

Comment:
I assume this is needed because the component could now be in any route and you need access to the list of scopes? 

Perhaps we should also update the comment for this service to also include now the lists instead of just the current active scope.

Reply:
Yes that is the reason.

Reply:
Yes, that is the reason behind this implementation, have a list of scopes (orgs + projects) available under any route.

Indeed, I miss the comment I will update it.